Command/MaintenanceCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Northern\MaintenanceModeBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+class MaintenanceCommand extends Command
+{
+    protected static $defaultName = 'app:maintenance';
```

Comment:
Would like to come up with a grouping for these, sort of like, `northern:maintenance` and then we can group commands into a `northern:` group!

Reply:
Done.